test/onnx/test_fx_to_onnx_with_onnxruntime_for_every_op.py

```diff
@@ -0,0 +1,275 @@
+            "argmax",
+            "__rmul__",
+            "special.ndtr",
+            "fft.ifftshift",
+            "nn.functional.mse_loss",
+            "mv",
+            "real",
+            "H",
+            "__rdiv__",
+            "fft.fftshift",
+            "nn.functional.silu",
+            "div",
+            "isnan",
+            "asin",
+            "mul",
+            "log1p",
+            "remainder",
+            "square",
+            "tan",
+            "logical_xor",
+            "nn.functional.soft_margin_loss",
+            "__rmod__",
+            "__radd__",
+            "mT",
+            "__rsub__",
+            "floor",
+            "sub",
+            "tanh",
+            "clamp_min",
+            "logical_or",
+            "positive",
+            "gt",
+            "expand_as",
+            "abs",
+            "round",
+            "msort",
+            "reciprocal",
+            "atan",
+            "log2",
+            "nn.functional.pairwise_distance",
+            "sqrt",
+            "reshape_as",
+            "nn.functional.celu",
+            "nn.functional.l1_loss",
+            "nn.functional.relu",
+            "eq",
+            "fliplr",
+            "true_divide",
+            "sign",
+            "rsub",
+            "log",
+            "nn.functional.feature_alpha_dropout",
+            "cos",
+            "contiguous",
+            "nn.functional.leaky_relu",
+            "nn.functional.softshrink",
+            "isinf",
+            "bmm",
+            "flipud",
+            "argmin",
+            "sin",
+            "add",
+            "float",
+            "kron",
+            "ge",
+            "pow",
+            "resolve_conj",
+            "maximum",
+            "nn.functional.embedding",
+            "conj",
+            "nn.functional.softsign",
+            "nn.functional.mish",
+            "outer",
+        }
+        target_dtype = torch.float
+        target_device = "cpu"
+
+        failed_op_names = set()
+        for op in self.op_db:
+            if op.name not in selected_op_names:
+                continue
+
+            samples = op.sample_inputs(target_device, target_dtype)
+            for sample_input in samples:
+                # Use the correct schema from get_signature_for_torch_op
+                # Then schema.bind(sample_input.input, *sample_input.args, **sample_input.kwargs).
+                args = [sample_input.input] + list(sample_input.args)
+                kwargs = sample_input.kwargs
+
+                if len(kwargs) > 0:
+                    continue
+
+                try:
+                    gm = functorch.make_fx(op)(*copy.deepcopy(args))
+                    self.run_test_with_fx_to_onnx_exporter(
+                        gm, args, rtol=1e-3, atol=1e-7
+                    )
+                except Exception as e:
+                    failed_op_names.add(op.name)
+
+        self.assertEqual(len(failed_op_names), 0, f"Failed op names: {failed_op_names}")
```

Comment:
I would parameterize the tests. Let me know if you need examples.

Reply:
Sure. Please give me some examples I can follow. Thanks.

Reply:
See if this one helps? https://github.com/microsoft/onnx-script/pull/208/files#diff-db423ee0dcd2d4a7c3dddb5141d8b3e2b4cc3f60b32354e36ec14ac3f61dbc01

You would simply need to replace the op name mapping with a list